index.html

```diff
@@ -4,64 +4,90 @@
-      src="./documentation/badge.html"
-      frameborder="0"
-      scrolling="no"
-      onload="resizeIframe(this)"
-    ></iframe>
-    <iframe
-      id="card-iframe"
-      src="./documentation/card.html"
-      frameborder="0"
-      scrolling="no"
-      onload="resizeIframe(this)"
-    ></iframe>
-    <iframe
-      id="image-iframe"
-      src="./documentation/image.html"
-      frameborder="0"
-      scrolling="no"
-      onload="resizeIframe(this)"
-    ></iframe> -->
-    <iframe
-      id="input-iframe"
-      src="./documentation/input.html"
-      frameborder="0"
-      scrolling="no"
-      onload="resizeIframe(this)"
-    ></iframe>
-    <script>
-      function resizeIframe(obj) {
-        obj.style.height =
-          obj.contentWindow.document.documentElement.scrollHeight + "px";
-      }
-    </script>
+    <header class="home-header">
+      <nav class="home-nav">
+        <a href="/"
+          ><img
+            class="home-logo"
+            src="./documentation/assets/logo.png"
+            alt="logo"
+        /></a>
+        <ul class="home-navlist">
+          <li class="home-navitem home-navitem-active">
+            <i class="fas fa-atlas"></i>
+            <a href="./documentation/documentation.html">Documentation</a>
+          </li>
+          <li class="home-navitem">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/humzahasan/FeedyUI/tree/development">
+              Source Code</a
+            >
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="home-main">
+      <section class="home-hero">
+        <div class="home-hero-left">
+          <img src="./documentation/assets/hero.svg" alt="hero-image" />
+        </div>
+        <div class="home-hero-right">
+          <h1>FeedyUI</h1>
+          <h3>
+            FeedyUI is an open-source, light-weight library of UI components
+            based on the vanilla CSS. It includes a vast varaities of classes to
+            support common html components used in day to day development.
+          </h3>
+        </div>
+      </section>
+      <section class="home-installation">
+        <h1 class="lg-title">Installation</h1>
+        <div>
+          <div class="home-installation-left">
+            <h3 class="mg-title">
+              In order to start using our component library, you need to include
+              the said lines in the head tag of your html page.
+            </h3>
+          </div>
+          <div class="home-installation-right">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=38px&ph=21px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Ffeedyui.netlify.app%252Flibrary%252Flib.css%2522%2520%252F%253E"
+              style="
```

Comment:
Just a suggestion : avoid inline css

Reply:
Good catch. I just started working on the documentation site. Will be adding common style for all iframe's.